CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
-## 19.0.0 (December 5, 2024\)
+## 19.0.0 (December 5, 2024)
 
 Below is a list of all new features, APIs, deprecations, and breaking changes. Read [React 19 release post](https://react.dev/blog/2024/04/25/react-19) and [React 19 upgrade guide](https://react.dev/blog/2024/04/25/react-19-upgrade-guide) for more information.
 
 > Note: To help make the upgrade to React 19 easier, we’ve published a react@18.3 release that is identical to 18.2 but adds warnings for deprecated APIs and other changes that are needed for React 19. We recommend upgrading to React 18.3.1 first to help identify any issues before upgrading to React 19.
 
 ### New Features
```

Comment:
Actually, I think the mistake is here. The changelog title is level 1 and then we skip level 2. "All Changes" below does it right and is at level 2. So I guess we need to dedent "New Features", "Breaking Changes", "Deprecations", "TypeScript changes", and "Notable Changes" instead.

That's what I did in the release: https://github.com/facebook/react/releases/tag/v19.0.0

Reply:
I see what you mean, done this in 4487a655821abbd59ebec85ba139feae6a7f0e4b.

We still have a somewhat incorrect document structure though, and it’s because version numbers have the same heading levels as _New features_, _All changes_. Here is the outline of `CHANGELOG.md`:

## `main`

<img width=300 src=https://github.com/user-attachments/assets/6065e796-dabc-46ea-8f0f-444183499cb6>

## This PR before review

<img width=300 src=https://github.com/user-attachments/assets/1ec4f4bd-f5f3-4517-891d-e815c8358bc8>

## This PR after 4487a655821abbd59ebec85ba139feae6a7f0e4b

<img width=300 src=https://github.com/user-attachments/assets/dcd53e8f-f355-4ea5-8484-70cf241f97a8>

So we can imagine software like Renovate bot scanning the mardown and detecting v19.0.0 changelog only between these lines:

https://github.com/facebook/react/blob/4487a655821abbd59ebec85ba139feae6a7f0e4b/CHANGELOG.md#L1-L7

As a permanent solution, we can switch to this in the whole markdown in a follow-up:

```markdown
# v1.2.3

## Optional group like _New Features_

### Area (e.g. React DOM Client)
```

Happy to open a separate PR if you see value in this kind of cleanup.

Reply:
Right, in the release it starts at level on and in the markdown at level 2